src/proteomes/__mocks__/proteomesEntryModelData.ts

```diff
@@ -0,0 +1,125 @@
+import { Rank } from '../../shared/types/apiModel';
+import {
+  CpdStatus,
+  ProteomesAPIModel,
+  ProteomeType,
+} from '../adapters/proteomesConverter';
+
+const mock: ProteomesAPIModel = {
+  id: 'UP000005640',
+  description: 'about some proteome',
+  taxonomy: {
+    scientificName: 'Homo sapiens',
+    commonName: 'Human',
+    synonyms: ['synonym'],
+    taxonId: 9606,
+    mnemonic: 'HUMAN',
+  },
+  modified: '2015-11-05',
+  proteomeType: ProteomeType.REDUNDANT,
```

Comment:
In other places, in order to be able to not have to use these, especially for big payloads where they would be complicated to notice, I switched to using unions of strings instead of an enum.

Reply:
To be fait, I found that using unions of strings didn't prevent TS to be able to help me find the possible values through tooling. If I'd start typing `proteomeType: '` VS code would come up with the list of possible values.

Reply:
Updated